src/main/java/cofh/thermaldynamics/duct/entity/Camera.java

```diff
@@ -17,7 +17,7 @@ public Camera() {
 		super(null);
 		this.width = 0;
 		this.height = 0;
-		invulnerable = true;
+		this.setEntityInvulnerable(true);
```

Comment:
context: Entity.invulnerable is private by default. To avoid the need for an AT, we can just use the setter.